### PR TITLE
Adding prow job for ocp-power-automation/ocp-power-automation.github.io

### PR DESCRIPTION
**Pull request by sudeeshjohn**

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/145#" title="Author self-approved">sudeeshjohn</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@sudeeshjohn: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pull-test-infra-yamllint | ffedcce8c98ff386272af3683d99c800b436c6ac | [link](https://prow.ppc64le-cloud.org/view/s3/prow-logs/pr-logs/pull/ppc64le-cloud_test-infra/145/pull-test-infra-yamllint/1368957255210242048) | `/test pull-test-infra-yamllint`

[Full PR test history](https://prow.ppc64le-cloud.org/pr-history?org=ppc64le-cloud&repo=test-infra&pr=145). [Your PR dashboard](https://prow.ppc64le-cloud.org/pr?query=is:pr+state:open+author:sudeeshjohn).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by ltccci** [comment]
@sudeeshjohn: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pull-test-infra-yamllint | be2636c5a6287ff6cf7d4b495d8bf0ff92476cf6 | [link](https://prow.ppc64le-cloud.org/view/s3/prow-logs/pr-logs/pull/ppc64le-cloud_test-infra/145/pull-test-infra-yamllint/1368958394299322368) | `/test pull-test-infra-yamllint`

[Full PR test history](https://prow.ppc64le-cloud.org/pr-history?org=ppc64le-cloud&repo=test-infra&pr=145). [Your PR dashboard](https://prow.ppc64le-cloud.org/pr?query=is:pr+state:open+author:sudeeshjohn).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: config/jobs/ocp-power-automation/ocp-power-automation.github.io/ocp-power-automation.github.io-postsubmits.yaml
Comment (mkumatag) [inline config/jobs/ocp-power-automation/ocp-power-automation.github.io/ocp-power-automation.github.io-postsubmits.yaml:13]: I just named the secret key as: `ocp-gh-deploykey`, can you please change the name here accordingly?

File: config/jobs/ppc64le-cloud/mkdocs/mkdocs-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/mkdocs/mkdocs-postsubmit.yaml:31]: before this following code needed, https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ppc64le-cloud/build-bot/buildbot-postsubmit.yaml#L36:L59 to generate a proper kubeconfig file to build the image

File: config/jobs/ppc64le-cloud/mkdocs/mkdocs-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/mkdocs/mkdocs-postsubmit.yaml:38]: This is not needed, will build only the x86 machine and run in the IKS cluster

File: config/jobs/ppc64le-cloud/mkdocs/mkdocs-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/mkdocs/mkdocs-postsubmit.yaml:17]: This is also not required

File: images/mkdocs/Dockerfile
Comment (mkumatag) [inline images/mkdocs/Dockerfile:12]: where is this used post image tagging

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: images/mkdocs/Dockerfile
Comment (sudeeshjohn) [reply images/mkdocs/Dockerfile:12]: Left unnoticed in the sample Dockerfile, removed

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/145#pullrequestreview-611976987" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/145#" title="Author self-approved">sudeeshjohn</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@sudeeshjohn: Updated the following 2 configmaps:
 * `plugins` configmap in namespace `prow` at cluster `default` using the following files:
   - key `plugins.yaml` using file `config/prow/plugins.yaml`
 * `job-config` configmap in namespace `prow` at cluster `default` using the following files:
   - key `ocp-power-automation.github.io-postsubmits.yaml` using file `config/jobs/ocp-power-automation/ocp-power-automation.github.io/ocp-power-automation.github.io-postsubmits.yaml`
   - key `mkdocs-postsubmit.yaml` using file `config/jobs/ppc64le-cloud/mkdocs/mkdocs-postsubmit.yaml`


<details>

In response to [this](https://github.com/ppc64le-cloud/test-infra/pull/145):

>


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
